File: V2EX/Main/RLTabbarVC.m
//
//  RLTabbarVC.m
//  V2EX
//
//  Created by ucs on 15/12/21.
//  Copyright © 2015年 ucs. All rights reserved.
//

#import "RLTabbarVC.h"


@interface RLTabbarVC ()

@end

@implementation RLTabbarVC

- (void)viewDidLoad {
    [super viewDidLoad];
    [self initUI];
    
    UINavigationController *nav1 = [[UINavigationController alloc] initWithRootViewController:[[RLTopicsTVC alloc] init]];
    nav1.tabBarItem.title = @"V2EX";
    [self addChildViewController:nav1];
    
    UINavigationController *nav2 = [[UINavigationController alloc] initWithRootViewController:[[RLNodesVC alloc] init]];
    nav2.tabBarItem.title = @"节点";
    [self addChildViewController:nav2];
    
}
//更改统一UI样式
- (void)initUI {
    UINavigationBar *bar = [UINavigationBar appearance];
    [bar setTintColor:V2EXGray];
}

@end

File: V2EX/Main/RLTabbarVC.swift
//
//  RLTabbarVC.swift
//  V2EX
//
//  Created by LLZ on 16/4/11.
//  Copyright © 2016年 ucs. All rights reserved.
//

import UIKit

class RLTabbarVC: UITabBarController {

    override func viewDidLoad() {
        super.viewDidLoad()
        initUI()
        
        let nav1 = UINavigationController.init(rootViewController: RLTopicsTVC())
        nav1.tabBarItem.title = "V2EX"
        self.addChildViewController(nav1)
        
        let nav2 = UINavigationController.init(rootViewController: RLNodesVC())
        nav2.tabBarItem.title = "节点"
        self.addChildViewController(nav2)
    }
    //更改统一UI样式
    private func initUI() {
        let bar = UINavigationBar .appearance()
        bar.tintColor = V2EXGray
    }

}
